### ISSUE TITLE
Adding a policy for blocking about:logins page

### ISSUE THREAD

**Issue by stoyianov** [mozilla/policy-templates#941]
So, currently there are several prefs/policies which can help restrict access to different elements of the saved logins function (and password manager as a whole) by users. Namely, `pref.privacy.disable_button.view_passwords`, `signon.showAutoCompleteFooter`, `DisablePasswordReveal` and `PasswordManagerEnabled`.
However, accessing the about:logins page is still possible, and from there - editing, copying passwords or exporting all of them in plaintext.

It's worth noting that policies such as `BlockAboutAddons`, `BlockAboutConfig`, `BlockAboutProfiles` and `BlockAboutSupport` which prevent access to the respective pages without touching their content/settings, with a message "Your organization has blocked access to this page", do exist.

I would say a similar policy, e.g. BlockAboutLogins would be of great service to many, including system administrators who aim to lock down workstations, specifically Firefox, as much as possible without additional software (in this case external password managers, etc.).

**Comment by mkaply** [comment]
You can use this policy to do that:

https://github.com/mozilla/policy-templates/blob/master/README.md#passwordmanagerenabled

**Comment by stoyianov** [comment]
Thanks for the quick reply!
Yeah, I also mentioned `PasswordManagerEnabled` - it does block about:logins, but thing is, it also removes functionality - disables password autofill, turns off prompts for saving login info and so on. Unlike any of the four `BlockAbout`s currently available, which simply block the page.

**Comment by mkaply** [comment]
If a user stores passwords, they would need to access those passwords.

I guess I don't understand the usecase for allowing people to store passwords, but not allowing them to change/remove/edit them?

**Comment by stoyianov** [comment]
It sounds weird on the surface, but from a certain point of view it makes a lot of sense. If we're talking about users' personal passwords, sure, of course, but otherwise not.

Say, a company has a list of websites it uses in day-to-day operations - vendor sites, email, government portals, registries, etc. - employees use shared company accounts for all of these, not individual accounts. So, the company needs to prevent them from even knowing the passwords, so that they can't share it with outsiders, or log in outside of the office, or from potentially infected devices, etc. Rules, NDAs, trust, password managers, 2FA in some cases - sure, all of this should work, but trust me, there are many smaller companies out there that do it this way and would rather rely on software restrictions if possible.

Anyway, I just came across a post from 2020, which I didn't find before posting myself - #661 - as described there, perhaps such a policy can be considered for about: pages in general.

**Comment by mkaply** [comment]
We have a disable password reveal policy which makes thing better:

https://github.com/mozilla/policy-templates/blob/master/README.md#disablepasswordreveal

Maybe a disable password copy might be useful here?

I assume you're disabling developer tools so a user can't simply remove the "Password" attribute from the entry field? And none of these sites have the ability to reveal the password?

**Comment by stoyianov** [comment]
That's right, dev tools need to be disabled and yes, from my experience almost all of the websites in question have no option to reveal what's in the password field.

Disable password copy would improve the current situation, that's correct.

Imagine you combine it with the rest of the policies I mentioned in my original post, including DisablePasswordReveal, the result would be View Saved Logins is unavailable in both Privacy settings and when clicking on fields with saved user/pass, and reveal/copy buttons are unavailable in about:logins, leaving only the export option.
